lomap/mcs.py

```diff
@@ -422,6 +424,12 @@ def getMapping(moli, molj, hydrogens=False, fname=None, time_out=150):
     ############ MCS BASED RULES ############
 
     # MCSR Rule
+    #sliu 0/17 add function to calculate tanimoto
+    def mtansr(self,):
```

Comment:
This is unused in the code. Remove? Or note that it is not used but is being retained in case you want to use it in the future?